microsoft-edge/webview2/how-to/webdriver.md

```diff
@@ -84,16 +84,16 @@ At this point you should have WebView2 Runtime installed, built a WebView2 proje
     using System.Threading.Tasks;
     ```  
     
-## Step 4: Drive WebView2 with Selenium and Microsoft Edge Driver  
+## Step 7: Drive WebView2 with Selenium and Microsoft Edge Driver  
 
 1.  First, create the `EdgeOptions` object, by copying the following code snippet.  
     
     ```csharp
     static void Main(string[] args)
     {
-        // EdgeOptions() requires using OpenQA.Selenium.Edge
-        // Construct EdgeOptions with is_legacy = false and the string "webview2"
-        EdgeOptions edgeOptions = new EdgeOptions(false, "webview2");
+
+        EdgeOptions edgeOptions = new EdgeOptions();
+
     ```  
     
     The `EdgeOptions` object takes in the following two parameters.  
```

Comment:
Should this be removed? Above the "EdgeOptions" class is no longer being created with parameters.

Reply:
We do still need an EdgeOptions object to configure in the next step. I'd suggest combining these two steps.

Reply:
I should clarify - we should still include and mention the EdgeOptions class. However, "takes in parameters" typically would refer to parameters or input to a constructor or function. In this case it's been changed to variables/properties on the object rather than arguments. I also don't see us using "is_legacy" of "webview2" string - instead these seem to be encapsulated with the "UseChromium" and "UseWebView" properties.